addons/website/static/src/js/backend/res_config_settings.js

```diff
@@ -33,4 +39,49 @@ BaseSettingController.include({
     },
 });
 
+const WebsiteCookiesbarField = FieldBoolean.extend({
+    _onChange: function () {
+        const checked = this.$input[0].checked;
+        if (!checked) {
+            return this._setValue(checked);
+        }
+
+        let message = $('<main/>', {
+            role: 'alert',
+        }).html(
+            $('<p/>').html(
+                _t("<b>Cookie bars may significantly impair the experience</b> of your visitors. We recommend you avoid them unless you have verified with a legal advisor that you absolutely need cookie consent in your country.")
+            )
+        ).append(_.str.sprintf(
+            _t("For session cookies, authentification and analytics, <b>you do not need to ask for the consent</b>, provided that your analytics is anonymized, which should be the case if you use Google Analytics (%s)."),
+            "see e.g. Opinion 04/2012 on Cookie Consent Exemption by the EU Art.29 WP"
+        ));
+
+        const cancelCallback = () => {
+            return this.$input[0].checked = !checked;
+        };
+        Dialog.confirm(this, null, {
+            title: _t("Please confirm"),
+            $content: $('<main/>', {
+                role: 'alert',
+            }).html(message),
+            buttons: [{
+                text: 'Do not activate',
+                classes: 'btn-primary',
+                close: true,
+                click: cancelCallback,
+            },
+            {
+                text: 'Activate bad experience',
```

Comment:
That button label sounds a bit childish to me. I'd rather not have the popup text at all either, but as we have it, why not make it more pro, e.g. "Activate anyway", or "Activate it despite the downsides" ?

Reply:
That one if a difficult one TBH, we don't want it either but FP insisted to have it exactly named that way.
I am kinda stuck between your review and FP's one. It has been more than a year battle to be allowed to add a cookies bar in Odoo, but it came with some compromises from my/our part, mainly this modal on res setting.

I will check with FP for `Activate it despite the downsides` as you suggest. No point to add it now and change it back to `Activate bad experience` once merged when he sees it hehe